docs/dist/pre-commit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+changed=$*
+if [[ "$changed" = "." ]];then
+    changed=$(find . -type f -name '*.php')
+fi
+if [[ -z "$changed" ]];then
+    changed=$(git diff --cached --name-only --diff-filter=ACM)
+fi
+if echo "$changed"|grep "\.php";then
+    CS_PHAR="docs/dist/phpcs.phar"
+    if [[ ! -e "$CS_PHAR" ]];then
```

Comment:
phpcs.phar can be replaced by /usr/bin/phpcs on Debian GNU Linux with package php-codesniffer installed.